templates/mail/auth/activate_email.tmpl

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
 	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-	<title>{{.Username}}, please verify your e-mail address</title>
+	<title>{{.Username}}, please verify your email address</title>
```

Comment:
I don't this change is needed

Reply:
I think it would be better as it was before

Reply:
Ok, no problem.